### ISSUE TITLE
Missing Name-Field entrys running "Tools - MailMerge --> RTF Document" in openCRX v5.2.2 (CRX)  

### ISSUE THREAD

**Issue by carusoe11** [opencrx/opencrx#126]
Generating RTF-documents for letter-labels  the content of the name-fields (FirstName and LastName) are missing. 

The fields for BusinessStreet, BusinessCity and BusinessPostalCode are printed out correctly. 

![rtf_export](https://github.com/opencrx/opencrx/assets/2705338/fea881a3-6f52-4ac1-bd82-6506c00345c6)

The MailMergeController.java shows only Iterators for PostalAddressLine and PostalStreet. The PostCode is transfered correctly.

![Bildschirmfoto vom 2023-07-29 08-14-50](https://github.com/opencrx/opencrx/assets/2705338/82d4b06f-3ce9-4903-b4e0-dc92c9bd12be)

**Comment by christoph-mueller-crixp** [comment]
please note that the address label is generated ENTIRELY from data available in the object PostalAddress - the reason why first/last name are missing in the labels you generated is most likely attributable to the fact that the attribute  postalAddressLine (labelled "Addresszeile(n)" in German) is empty - see the following screenshot for an example with populated and empty postalAddressLine:
![postalAddressLine](https://github.com/opencrx/opencrx/assets/4585184/f5c06196-3842-4aa4-88bc-65106b7d65f3)


To summarize: The object PostalAddress should be populated with the data that as you need it on a postal address label, either manually or based on an algorithm (that you can implement as a wizard, for example)

**Comment by carusoe11** [comment]
Using the very helpfull hints given above the feature runs without any problems. - Perfect. 
Excusions for causing a false alarm....
